src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --blue-color: #00A7E1;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+}
```

Comment:
Avoid setting margins and paddings on the `body` element directly. Instead, use a container element inside the `body` to apply these styles, to maintain consistency and avoid potential issues with collapsing margins.